2018/proposal.md

```diff
@@ -0,0 +1,100 @@
+PyCon AU 2018 Education Seminar Proposal
+========================================
+
+If accepted, the PyCon Australia Education Seminar will bring together educators who are using Python
+in teaching or looking for new ways to use coding in education. The goal is to share stories, strategies
+and techniques for engaging students and teaching with Python that can be directly applied by other educators. 
+
+Python has long had reputation as an effective beginner programming language. Bringing together educators and
+developers helps developers provide better support for beginners in Python and its libraries, as well as
+giving educators the opportunity to develop their own technical skills further.
+
+As in previous years, talk topics at the seminar will be open to any educational use of Python but the core
+focus will continue to be on primary and secondary school education. This includes both formal classroom
+instruction and extracurricular activities.
+
+However, some changes and additions are also being proposed based on our experiences with previous iterations
+of the event.
+
+Event Organisers
+----------------
+
+* Nick Coghlan
+  * 2015 Education Seminar founder & coordinator
+  * 2016/17 Education Seminar co-coordinator with Katie Bell
+  * CPython core developer
+
+* Dr Nicky Ringland
+  * Computing Education Specialist at the Australian Computing Academy
+  * Outreach Officer at the National Computer Science School
+  * Co-founder of Grok Learning
+
+* Additional organisers:
+  * Amanda Hogan (Australian Computing Academy)
+  * Bruce Fuda (Australian Computing Academy)
+
+Event Outline
+-------------
+
+Prior to the afternoon tea break, the 2018 Education Seminar will follow the same
+structure as in previous years: an invited speaker prior to morning tea, 3 speakers
+from the Call for Proposals prior to lunch, then 3 more speakers prior to afternoon
+tea.
+
+For the final session, we'd like to work with the main conference organisers to
+arrange to run a "Student Showcase", inspired by the well-received student
+presentations at PyCon Australia 2017.
+
+The general idea of this part of the proposal would be to:
+
+* Provide a way for teachers & senior secondary students to nominate as potential
+  presenters at the Showcase (similar to the way Gungahlin College students did
+  through the regular CfP for 2017)
+* Accept up to 6 projects for presentation (these would each be given a 15 minute
```

Comment:
I just remembered that this math doesn't quite work, since the sessions are normally structured as 30+10+30+10+30=110 minutes, rather than 120 minutes (since we only need the 10 minute changeover either side of the middle talk, not before the first talk or after the third one).

Still, I think 5 or 6 shorter presentations will be a better option here than 3 regular length ones.

Reply:
Yup. I don't think we'll need 5 minutes change between each talk, either. We'll get the kids' slides in advance, put them all on one computer, deal with code demos as required... we can cut that down pretty easily. :)

Reply:
And use a handheld microphone to avoid the need to get lapel mikes sorted out.

Reply:
I made the math work by cutting the nominal transition time to 3 minutes. That gives a nominal 105 minutes total. Whatever we actually do in practice will need to be coordinated with the AV team anyway (since they need to know how to cut the videos appropriately), but I think this is fine for the proposal stage.